tdrs-backend/tdpservice/scheduling/db_backup.py

```diff
@@ -142,7 +142,7 @@ def restore_database(file_name, postgres_client, database_uri, system_user):
         logger.info(msg)
     except Exception as e:
         logger.error(f"Caught exception while creating the database. Exception: {e}.")
-        return False
+        raise e
```

Comment:
Why bother with the try/except if we're going to raise anyway? The logging line doesn't currently add value to the troubleshooting process.

Reply:
Mostly for consistency with the rest of the functions in this file. Every other function in the file follows that needs exception handling follows that pattern. Also, most of those functions are called from different processes so it helps a little with traceability imo.